supplemental/arithmetic/cosh.md

```diff
@@ -0,0 +1,62 @@
+# Cosh
+
+## Options
+
+### Rounding
+
+Hyperbolic cosine of an input can yield a result that is not exactly
```

Comment:
Here you don't capitalize cosine (I prefer this form) and for all the others you did.  Could we capitalize the names consistently in one form or the other?

Reply:
made the change, thanks.